.github/workflows/build_v5_screener_imgs-test.yml

```diff
@@ -0,0 +1,46 @@
+name: ECR Push v5 (experimental)
```

Comment:
I'm confused. Shouldn't this workflow be replacing our current v5 workflow: https://github.com/Contrast-Security-OSS/NodeTestBenches/blob/main/.github/workflows/build_v5_screener_imgs.yml and then rename it here and in the file name?

Reply:
Yep, it's just for testing purposes here.

Reply:
Deleted

Reply:
Oh so you just need me to see if I can build these images locally?